kesconf/file.go

```diff
@@ -298,6 +298,9 @@ type LogConfig struct {
 	// Audit determines whether the KES server logs audit events to STDOUT.
 	// It does not en/disable audit logging in general.
 	AuditLevel slog.Level
+
+	// Log level for which to report KES diagnostic messages.
+	LogLevel slog.Level
```

Comment:
I don't think we need another log-level field.

Currently you can set the `ErrorLevel` to:
```
off
on
debug
info
warn
error
```

It should be sufficient to set:
```
log:
  error: DEBUG
```